core/python_utils.py

```diff
@@ -218,6 +219,23 @@ def open_file(filename, mode, encoding='utf-8', newline=None):
         raise IOError('Unable to open file: %s' % filename)
 
 
+def get_package_file_contents(package: str, filepath: str) -> str:
+    """Open file and return a corresponding file object.
```

Comment:
Explain more about why this is "package"-specific -- the docstring makes it sound like this works for any file. Try to help developers understand what they can and cannot do with this.

Reply:
Done.